eng/scripts/Language-Settings.ps1

```diff
@@ -6,25 +6,32 @@ $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
 
+if ((Get-Command npm | Measure-Object).Count -eq 0 ) 
+{
+  LogError "Could not locate npm. Install NodeJS (includes npm and npx) https://nodejs.org/en/download"
+  exit 1
+}
+
 function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
 {
   $projectPath = Join-Path $pkgPath "package.json"
   if (Test-Path $projectPath)
   {
     $projectJson = Get-Content $projectPath | ConvertFrom-Json
     $jsStylePkgName = $projectJson.name.Replace("@", "").Replace("/", "-")
-    if ($pkgName -eq "$jsStylePkgName" -or $pkgName -eq $projectJson.name)
+    if ($pkgName -and ($pkgName -ne $jsStylePkgName -or $pkgName -ne $projectJson.name))
```

Comment:
Shouldn't this condition be "-and" instead of "-or"
if ($pkgName -and ($pkgName -ne $jsStylePkgName **-or** $pkgName -ne $projectJson.name))

Reply:
Yes I think your correct that this should be an and. 
```suggestion
    if ($pkgName -and ($pkgName -ne $jsStylePkgName -and $pkgName -ne $projectJson.name))
```